buildtools/src/main/resources/bookkeeper/server-suppressions.xml

```diff
@@ -36,7 +36,11 @@
     <suppress checks=".*" files=".*[\\/]streaming[\\/].*"/>
     <suppress checks=".*" files=".*[\\/]tools[\\/].*"/>
     <suppress checks=".*" files=".*[\\/]util[\\/].*"/>
+    <suppress checks=".*" files=".*[\\/]tls[\\/].*"/>
```

Comment:
why do you suppress these packages? I think those packages are already under checkstyle validation, no?

Reply:
@sijie I think that in bookkeeer-server we do not have checkstyle at all

see the current pom.xml, checkstyle is to be enabled explicitly until we cover the whole code base :
https://github.com/apache/bookkeeper/blob/master/bookkeeper-server/pom.xml

in "commons" module we have checkstyle
https://github.com/apache/bookkeeper/blob/master/bookkeeper-common/pom.xml

With this change I am enabling checkstyle ONLY for client.api package

maybe I am missing something

Reply:
I think @jiazhai was adding this file at commit f581e030c79abbde81bf165603fccdb5e6a1d38a I think he added for the purpose to enable checkstyle on bookkeeper-server. If this doesn't enable, I would suggest having a separate issue to enable this checkstyle first.